hibernate-cache/src/main/resources/org/hibernate/cache/infinispan/builder/infinispan-configs-local.xml

```diff
@@ -17,20 +17,22 @@
 
       <!-- Default configuration is appropriate for entity/collection caching. -->
       <local-cache-configuration name="entity" simple-cache="true" statistics="false" statistics-available="false">
+         <locking isolation="READ_COMMITTED"/>
```

Comment:
True.

However, there's a problem with `QueryRegionImplTest`. That test runs with transactional caches and the default isolation level is the RR. When running with RR, the test breaks. That's why I forced RC in the configurations.

What would be the correct way to solve this? I went for the option of setting isolation to RC everywhere, but really it was only needed for `replicated-query` cache.

Maybe the test should run in non-transactional mode?